src/styles/_stars.scss

```diff
@@ -0,0 +1,17 @@
+.stars {
+  display: flex;
+
+  &__star {
+    background-image: url('../images/star.svg');
+    background-repeat: no-repeat;
+    background-position: center;
+    width: 16px;
+    height: 16px;
+    border-radius: 0.5px;
```

Comment:
The 'border-radius' property seems to be unnecessary here. The star images provided do not have rounded corners, so this property doesn't affect the appearance of the stars. If there's no specific reason for it, you may want to remove it.